lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py

```diff
@@ -211,7 +211,7 @@
         target=dict(type='str', required=True, aliases=['entry'])
     ),
     TXT=dict(
-        value=dict(type='str', required=True, aliases=['entry'])
+        value=dict(type='list', required=True, aliases=['entry'])
```

Comment:
Please consider the compatible issue. How about add a TXTs field, and construct TXTs if user continue uses TXT. And then please also modify the documentation at the beginning of this python file, together with an deprecation warning for the TXT. You can refer the `networkinterface` module to see the deprecation warning

Reply:
@yuwzho thank you for the review. If I understand you correct, you are talking about creating a new field and leaving the existing behaviour or TXT intact? If that is the case, I do not think that is required as the current implementation will not work in any case, as this will fail against the Azure RM API. In addition the usage syntax has not changed and remains the same. With that in mind, do you think it is still required to create a new field?

Note that this change would still retain the behaviour where a `string` is provided as the value to the `entry`.

Reply:
It OK, is there any test cover the old type?

Reply:
There weren't any test cases for TXT records prior to this commit. The first and second records in the provided test case are of the old value types (str).